morituri/program/flac.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, CalledProcessError
+
+import logging
+logger = logging.getLogger(__name__)
+
+FLAC = 'flac'
+
+
+def encode(infile, outfile):
+    """
+    Encodes infile to outfile, with flac.
+    Uses '-f' because morituri already creates the file.
+    """
+    try:
+        check_call(['flac', '--totally-silent', '--verify', '-o', outfile,
+                    '-f', infile])
```

Comment:
Hum, could we get compression option configuration? I’d like to compress with --best, and unless we make it the default, this will require a config option. More generally, I suppose people might want to change flac call parameters, so having it in config would be great. ;)